src/content/docs/ko/guides/authentication.mdx

```diff
@@ -188,6 +189,162 @@ const session = await getSession(Astro.request);
 - [GitHub의 `auth-astro`](https://github.com/nowaythatworked/auth-astro?tab=readme-ov-file#auth-astro)
 - [Auth.js 문서](https://authjs.dev/)
 
+### Next Steps
```

Comment:
Pointing out this heading is unstranslated, not sure if it's intentional!

Reply:
Thank you!